src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -2702,6 +2702,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             Return If(IOperationFeatureFlag Is Nothing, False, options.Features.ContainsKey(IOperationFeatureFlag))
         End Function
+
+        Friend Overrides Function IsUnreferencedAssemblyIdentityDiagnosticCode(code As Integer) As Boolean
+            Select Case code
+                Case ERRID.ERR_UnreferencedAssemblyEvent3,
```

Comment:
>ERR_UnreferencedAssemblyEvent3 [](start = 27, length = 30)

It looks like there is no targeted test for this error, consider adding verification to Error_Regress40025DLL unit-test.